scipy/stats/_stats.pyx

```diff
@@ -602,6 +602,15 @@ cdef double _studentized_range_pdf(int n, double[2] integration_var,
     return math.exp(log_terms) * math.pow(_Phi(s * q + z) - _Phi(z), k - 2)
 
 
+cdef double _studentized_range_pdf_asymptotic(double z, void *user_data) nogil:
+    # evaluates the integrand of equation (2) by Lund, Lund, page 205. [4]
+    # destined to be used in a LowLevelCallable
```

Comment:
Note quite, right? This is the PDF, so it's the derivative of the integrand w.r.t. Q, no? And we actually worked from the formula shown on Wikipedia, which is written a little differently, so it might help to explain how this is equivalent if equation 2 is the reference.